downstream/modules/platform/ref-controller-run-the-builder.adoc

```diff
@@ -0,0 +1,43 @@
+[id="ref-controller-run-the-builder"]
+
+= Run the builder
+
+The `ansible-builder build` command takes an {ExecEnvShort} definition as an input. 
+It outputs the build context necessary for building an {ExecEnvShort} image, and then builds that image. 
+The image can be re-built with the build context elsewhere, and produces the same result. 
+By default, it searches for a file named `execution-environment.yml` in the current directory.
+
+The following example `execution-environment.yml` file can be used as a starting point:
+
+[literal, options="nowrap" subs="+attributes"]
+----
+---
+version: 3
+dependencies:
+  galaxy: requirements.yml
+----
+
+The content of `requirements.yml`:
+
+[literal, options="nowrap" subs="+attributes"]
+---
+collections:
+  - name: awx.awx
+----
+
+To build an {ExecEnvShort} using the files above, run:
+
+[literal, options="nowrap" subs="+attributes"]
+---
+$ ansible-builder build
```

Comment:
Would it make sense to keep the command and the output separate, so that users can copy the command?
That said, it's a short command, so it mightn't be worth the trouble.